internal/gatewayapi/envoyextensionpolicy.go

```diff
@@ -534,12 +534,11 @@ func (t *Translator) buildExtProc(
 	return extProcIR, err
 }
 
-func irConfigNameForEEP(policy *egv1a1.EnvoyExtensionPolicy, idx int) string {
+func irConfigNameForExtProc(policy *egv1a1.EnvoyExtensionPolicy, index int) string {
 	return fmt.Sprintf(
-		"%s/%s/%d",
-		strings.ToLower(egv1a1.KindEnvoyExtensionPolicy),
-		utils.NamespacedName(policy).String(),
-		idx)
+		"%s/extproc/%s",
```

Comment:
ext-proc instead ? cc @guydc

Reply:
EG is using flattened names like "envoyextensionpolicy", and envoy is using "snake_case".